pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -315,6 +315,36 @@ func (b *awsElasticBlockStoreMounter) SetUpAt(dir string, fsGroup *int64) error
 	return nil
 }
 
+// AwsEbsMountErrorHint performs some basic analysis
+// on the current mount error returned from the plugin
+// and will add a user hint or resolution tip for enhanced UXP
+// If no matches then original error is returned
+func AwsEbsMountErrorHint (inErr error) error {
```

Comment:
Why not use a special error type for this and implement `String` for formatting?